.tx/config

```diff
@@ -8,6 +8,13 @@ source_lang = en
 type = STRINGS
 lang_map = pt_BR:pt-BR,zh_CN:zh-Hans,fi_FI:fi-FI,es_MX:es-MX,nb_NO:nb-NO,cs_CZ:cs,en_GB:en-GB,es_AR:es-AR,sk_SK:sk-SK,hu_HU:hu,ka_GE:ka-GE,zh_TW:zh-Hant-TW,es_CL:es-CL,es_CO:es-CO,es_CR:es-CR,es_DO:es-DO,es_EC:es-EC,es_GT:es-GT,es_HN:es-HN,es_NI:es-NI,es_PA:es-PA,es_PE:es-PE,es_PR:es-PR,es_PY:es-PY,es_SV:es-SV,es_UY:es-UY,es_419:es-419,pt_PT:pt-PT,ja_JP:ja
 
+[nextcloud.talk-ios-plurals]
```

Comment:
@rakekniven new resource for iOS talk plurals. Once this is confirmed working (and it seems to work at the moment), we will also do a -plurals for the files app.
Can you add it to your forum post list?